lib/testmap.py

```diff
@@ -304,7 +304,8 @@ def tests_for_image(image):
 def tests_for_po_refresh(project):
     if project == "cockpit-project/cockpit":
         return [TEST_OS_DEFAULT]
-    return REPO_BRANCH_CONTEXT.get(project, {}).get("master", [])
+    branches = REPO_BRANCH_CONTEXT.get(project, {})
```

Comment:
and then this could also use `get_default_branch()` instead of guessing between master and main.